schemas/dublin-core/src/main/plugin/dublin-core/index-fields/index.xsl

```diff
@@ -153,10 +192,15 @@
 
       <xsl:variable name="tags"
                     select="dc:subject[. != '']"/>
+
+      <tagNumber>
+        <xsl:value-of select="count($tags)"/>
+      </tagNumber>
+
       <xsl:if test="count($tags) > 0">
         <tag type="object">[
           <xsl:for-each select="$tags">
-            {"default": <xsl:value-of select="concat($doubleQuote, gn-fn-index:json-escape(.), $doubleQuote)"/>}
```

Comment:
You need to escape the string no ? test with a keyword like `Code "44"`

Reply:
Yep - keyword in error.  I have introduced an small update to fix it. 
![Screen Shot 2021-02-12 at 16 13 04](https://user-images.githubusercontent.com/576292/107785721-3a1fbd00-6d4d-11eb-9320-13d3fb688d41.png)